### PR TITLE
Game Piece Scoop

### PR DESCRIPTION
**Pull request by BreadBurns**
Finished most parts of the subsystem
Currently is set to use a CANSparkMax

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/GamePieceScoop.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.Servo;
+import edu.wpi.first.wpilibj.motorcontrol.MotorControllerGroup;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class GamePieceScoop extends SubsystemBase {
+    /** Creates a new GamePieceLift. */
```
Comment (PNovak619) [inline src/main/java/frc/robot/subsystems/GamePieceScoop.java]: please change comment to refer back to subsystem name

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -40,5 +40,7 @@ public final class Constants {
     public static final MotorType UPRIGHT_MOTOR_TYPE = null;
     // TODO: set this to something else than 0
     public static final int LED_PWM_ID = 0;
+    public static final int GAME_PIECE_LIFT_MOTOR_2 = 0;
```
Comment (PNovak619) [inline src/main/java/frc/robot/Constants.java]: please change all game piece lift names to gamepiece scoop

File: src/main/java/frc/robot/subsystems/GamePieceScoop.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+    private Servo gamePieceLiftMotor1;
```
Comment (PNovak619) [inline src/main/java/frc/robot/subsystems/GamePieceScoop.java]: please change all game piece lift names to game piece scoop

File: src/main/java/frc/robot/subsystems/GamePieceScoop.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    private Servo gamePieceLiftMotor2;
+    private MotorControllerGroup gamePieceLiftMotorGroup;
+    // TODO: Figure out names for the motors
+    // TODO: Confirm the types of motors to be used
+    public GamePieceScoop() {
+        gamePieceLiftMotor1 = new Servo(Constants.GAME_PIECE_LIFT_MOTOR_1);
+        gamePieceLiftMotor2 = new Servo(Constants.GAME_PIECE_LIFT_MOTOR_2);
+        }
+
+    public void liftServo() {
```
Comment (PNovak619) [inline src/main/java/frc/robot/subsystems/GamePieceScoop.java]: might be better to change to servo on and servo off?

**Review by PNovak619 — APPROVED** [review]
Nice job

**Review by programmer94 — APPROVED** [review]

**Pull request merged**